### DISCUSSION TITLE
访问添加了 http basic authentication，无法正确弹框

### DISCUSSION THREAD

**Discussion by cloudGrin** [tw93/Pake#331] [Q&A]
在浏览器端访问网页时会进行弹框，让输入密码，如图1。
Pake打包后无法弹框，如图2。
<img width="1107" alt="image" src="https://user-images.githubusercontent.com/27754342/223013041-0c0a5629-19ce-4407-afde-e89fd4b6a96b.png">

<img width="1183" alt="image" src="https://user-images.githubusercontent.com/27754342/223012915-d64dfd37-16d3-45dd-b28d-9fc8f5c88755.png">

暂时通过这个方法通过了认证 [https://username:password@www.example.com/](https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication#access_using_credentials_in_the_url)

**Comment by aiens** [comment]
我尝试了[https://username:password@www.example.com/] 打包出来的dmg，依旧401。

**Reply by tw93** [reply]
的确现在不支持系统浏览器方式的弹框，这里建议试试菜单栏里面任意地址跳转的功能
